README.md

```diff
@@ -1,11 +1,36 @@
 # OpencC2Test
 
+OpenC2 is a standardized language for controlling cyber defense technologies. It allows different systems and components to communicate and coordinate with each other, regardless of the specific products or technologies they use. This enables automated and efficient management of cyber defenses.
+
+## Setup Guide
+
 To start your Phoenix server:
 
   * Run `mix setup` to install and setup dependencies
+  * Source the environment variables by running `source .env`. This contains environment specific settings and keys.
```

Comment:
Add instruction to add the `.env` and the environment variables to be added .

Reply:
I have it like this but is the password also a public thing or not?

* Create a file `.env` in the project root and add the following environment variables;
    * export CLIENT_ID=:add your client_id
    * export MQTT_HOST=test.mosquitto.org
    * export MQTT_PORT=1883
    * export USER_NAME=plug
    * export PASSWORD=fest